doc/packer.txt

```diff
@@ -563,4 +563,8 @@ invoked as follows:
   }
 - With a list of plugins specified in either of the above two forms
 
+For the *cmd* option, the command may be a pattern instead of an explicit pattern. If the command contains any
```

Comment:
I think this is a bit confusing, i.e. `the command maybe a pattern instead of an explicit pattern`. I think maybe saying that it could be the full command or a lua pattern is clear enough 🤷🏿‍♂️

Reply:
Should have clarified the bit about the alphanumeric characters I think is fine just the pattern/explicit pattern comment